_posts/2021-03-09-gsoc2021.md

```diff
@@ -0,0 +1,118 @@
+MDAnalysis 2021"
+style="float: left; height: 4.5em; " />
+</p>
+
+MDAnalysis has been accepted as an [organization][org] into [Google Summer of Code 2021][gsoc]!
+If you are a student who is interested in working with us
+this summer, please read the advice and links below and write to us on the
+[mailing list].
+
+We are looking forward to all applications from interested students
+(*undergraduate* and *postgraduate*).
+
+The application window **deadline** is **April 13, 2021 11:00 (Mountain Standard Time)**. As
+part of the application process you must familiarize yourself with [Google
+Summer of Code 2021][gsoc]. 
+
+If you are interested in working with us please read on and contact us on our [mailing list].
+[Apply] as soon as possible; the application window opens on March 29, 2021.
+
+# Project Ideas
+
+If you have your own idea about a potential project we'd love to work
+with you to develop this idea; please write to us on the [developer
+list][mailing list] to discuss it there.
+
+We also have listed several [possible projects][ideas] for you to work
+on, on our wiki. Our initial list of ideas (see summaries in the
+table below) contains 6 projects at different levels of difficulties
+and with different skill requirements. However, check the [ideas][]
+page — we might add more ideas after the posting date of this post.
+
+You don't need to have all the skills that we are listing, although
+that helps, of course. But you need to demonstrate to us that you're
+able and keen to learn anything that you don't know yet, and we will
+be happy to help you learn during your project with us.
+
+| project | name                                                                                                                                                      | difficulty | description                                                                                    | skills                      | mentors                                |
+|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|------------|------------------------------------------------------------------------------------------------|-----------------------------|----------------------------------------|
+| 1       | [Molecular volume and surface analysis]({{ site.github.wiki }}/Project-Ideas-2021#project-1-molecular-volume-and-surface-analysis) | easy       | use an existing package for molecular surface area calculations to build a new analysis module | Python, MDAnalysis.analysis | @orbeckst, @IAlibay  |
+| 2       | [AtomGroupGroup and Rings]({{ site.github.wiki }}/Project-Ideas-2021#project-2-atomgroupgroup-and-rings)                           | medium     | Write analysis module for pi stacking analysis                                                 | Python                      | @richardjgowers, @micaela-matta, @IAlibay        |
```

Comment:
Current discussion on discord would be to drop project 2 and have project 3 take precendence instead.